backend/engine/plugins/github_repo_health/cli/src/rules/helpers.py

```diff
@@ -25,16 +37,22 @@ def evaluate_array_config(config, eval_fn):
 
 
 def add_metadata(passing, check, config={}, error_message=None):
+    id = config.get("id") or check.identifier
     name = config.get("name") or check.name
     description = config.get("description") or check.description
+    severity = config.get("severity")
```

Comment:
Should this have a default value?

Reply:
I don't think so, unless we explicitly want to tell it to be `None`.

The intent is for `severity` to only show up in the output of repo health if it's specified in the config